packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/index.tsx

```diff
@@ -308,10 +310,15 @@ class TransactionsContainer extends React.PureComponent<Props> {
 }
 
 const mapStateToProps = (state, ownProps: OwnProps): LinkStatePropsType =>
-  getData(state, ownProps.coin, ownProps.coinfig)
+  getData(
+    state,
+    ownProps.computedMatch.params.coin,
+    window.coins[ownProps.computedMatch.params.coin].coinfig
```

Comment:
Potentially unsafe

Reply:
Previously `window.coins` and `coin` props were passed in from app.js, but were still essentially doing this same thing. I could pass in window.coins from app.js but not the coin because we get it from react-router now. I could also get the coinfig in selectors.ts and add some checks to see if it exists or else throw.